docs/pages/introduction/why-not-expo.mdx

```diff
@@ -4,107 +4,40 @@ sidebar_title: Limitations
 ---
 
 import { InlineCode } from '~/components/base/code';
-import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
 
-Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in the decisions they make.
+We believe Expo tools are a great option for anyone looking to create amazing cross-platform apps quickly. Nonetheless, everyone's project is unique, and there may be reasons Expo is not the right choice for what you're building. Read on for some reasons why you may _not_ want to use Expo tools.
 
-<Collapsible summary="Used Expo before? Here are some recent changes to the managed workflow you should be aware of…">
+## EAS
 
-With the release of Expo Application Services, the managed workflow has far fewer limitations, and can be used for any project no matter the complexity:
+For more information on what current limitations exist with EAS, see the following:
 
-- All libraries available to React Native apps are available to Expo managed workflow apps built with [EAS Build](/build/setup), but _some_ may require a [Prebuild Config Plugin](/guides/config-plugins/) to be added. The addition of config plugins means:
-  - Any 3rd party push notification service can support managed workflow apps.
-  - Native libraries that integrate with proprietary services can support managed workflow apps.
-- Apps built with EAS Build can be as small as you want since the entire Expo SDK is no longer bundled in by default.
-- Apps built with EAS Build can target children under 13 years old.
+<BoxLink
+  title="EAS Build"
+  description="Some of the missing features from EAS could prevent you from being able to use the service for your applications. Learn more."
```

Comment:
```suggestion
  description="EAS Build is designed to work for any React Native project, whether or not you also use Expo Prebuild or manage your own native files. In some situations, you may need to change your project configuration, or it may be incompatible with your app. Learn more."
```
Since this will get quoted, want to make sure we're leading with "it should work with just about anything, but check this stuff to be sure". Also not big on the "missing features" verbiage (we may want to update on the EAS Build limitations page, too), since some of the edge cases in there don't really indicate something that's missing, per se.